logdevice/server/ServerSettings.cpp

```diff
@@ -590,6 +590,29 @@ void ServerSettings::defineSettings(SettingEasyInit& init) {
      "If disabled, server will use default DSCP value ",
      SERVER | REQUIRES_RESTART,
      SettingsCategory::Network)
+
+     ("enable-automatic-traffic-class-selection",
+     &enable_automatic_traffic_class_selection,
+     "false",
+     nullptr,
+     "If enabled, if client is "
+     "If disabled, server will use default DSCP value ",
+     SERVER,
+     SettingsCategory::Configuration)
+
+    ("traffic-class-updation-threshhold",
+     &traffic_class_updation_threshhold,"1800s",
+     [](std::chrono::milliseconds val) -> void {
+       if (val.count() <= 0) {
+         throw boost::program_options::error(
+           "traffic-class-updation-threshhold must be positive"
+         );
+       }
+     },
+     "If enabled, traffic classes of read stream are update automatically "
+     "on the basis of how much they have caught up",
+     SERVER,
+     SettingsCategory::Configuration)
```

Comment:
Can we make it one setting automatic-traffic-class-selection-threshold and make 0 as disabled?

Reply:
done